independent-publisher-connectors/Proposal - GIPHY (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,848 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GIPHY",
+    "description": "This custom connector wraps around GIPHY API. It can be used to get GIFs related to a search string that is provided as input",
+    "version": "1.0"
+  },
+  "host": "api.giphy.com",
+  "basePath": "/v1/gifs/search",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/": {
+      "get": {
+        "responses": {
+          "default": {

```

Comment:
@priyanshu870 default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

Reply:
Done